.github/workflows/pullrequest-stale.yml

```diff
@@ -0,0 +1,25 @@
+name: Stale
+on:
+  schedule:
+    - cron: "0 * * * *"
+
+jobs:
+  stale:
+    if: github.repository_owner == 'ls1intum'
+    runs-on: ubuntu-latest
+    steps:
+      - name: 7 days stale PRs
+        uses: actions/stale@v5
+        with:
+          days-before-stale: 7
+          days-before-close: 14
+          # Disable issue checking, only PR
+          days-before-issue-stale: -1
+          remove-stale-when-updated: true
+          stale-pr-label: "stale"
+          exempt-pr-labels: "no-stale"
+          stale-pr-message: >
+            There hasn't been any activity on this pull request recently. This
+            pull request has been automatically marked as stale because of that
+            and will be closed if no further activity occurs within 7 days.
+            Thank you for your contributions.
```

Comment:
Ran this message through Grammarly and this is what is suggested. 
> There hasn't been any activity on this pull request recently. Therefore, this pull request has been automatically marked as stale and will be closed if no further activity occurs within seven days. Thank you for your contributions.

You don't really have to change this, but I just wanted to let you know. (If you do write out the 7 to seven, then it might make sense to make it bold by writing `**seven**`)

Reply:
Yes that sounds better, I updated it!